support/development/prepareLocalDev.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+set -euo pipefail
+
+function show_step(){
+    echo "-------------------------------------------"
+    echo "Running step: ${1}"
+    echo "-------------------------------------------"
+}
+
+show_step 'Remove existing test and temp output.'
+find . -name '.nyc_output' -type d -prune -not -path "*/node_modules/*" -exec rm -r '{}' '+'
+find . -name 'coverage' -type d -prune -not -path "*/node_modules/*" -exec rm -r '{}' '+'
+find . -name '.tmp' -type d -prune -not -path "*/node_modules/*" -exec rm -r '{}' '+'
+
+show_step 'Install npm packages for core.'
+cd core
+npm install
+node_modules/.bin/tsc
+cd ..
+
+show_step 'Compile core.'
+npm install
+node_modules/.bin/tsc
+
+show_step 'Compile frontend.'
+sudo rm -rf ../portal-ng-form-custom
+npm run compile:ng
+npm run webpack
+
+show_step 'Build api descriptors.'
```

Comment:
Not really needed for dev, this script is used in the CI process to generate the API documentation

Reply:
I'll remove the build api descriptors step. I didn't realise it was a more involved process.